src/components/back-button/index.tsx

```diff
@@ -8,8 +8,9 @@ interface Props {
 
 export const BackButton: React.FC<Props> = (props) => {
   return (
-    <button className="btn-back" onClick={props.onTap}>
-      <CaretLeft size={20} />
-    </button>
+    <S.BackButton onClick={props.onTap}>
```

Comment:
mesma coisa do comentário do @racsandoval 
```suggestion
    <BackButton onClick={props.onTap}>
```

Reply:
Corrigido! Já corrigi em todos os arquivos. Valeu Beto!